### DISCUSSION TITLE
Service Code Requirement in allowed_amounts File

### DISCUSSION THREAD

**Discussion by APWU-GRoberts** [CMSgov/price-transparency-guide#321] [Q&A] [answered]
I have been reviewing the 0.8.0 version of the allowed_amounts file and noticed the service_codes field has been updated.  The field is marked as required only when the billing_class is "professional".  Additionally, the description of the field states "...placed on a professional claim to indicate the setting in which a service was provided."  I read this to mean that the service_code should only be present on "professional" claims.  Is this correct?

Can a sample "institutional" entry be added to the allowed_amounts example file?

**Comment by shaselton-usds** [comment, marked as answer]
@APWU-GRoberts There are scenarios in which you may have contracts that leverage `service_codes` for "institutional" `billing_class` values. If that is the case for your organization, you should include them. Otherwise, they are only required for "professional" `billing_class` arrangements.

**Reply by Rashmi-sb** [reply]
@shaselton-usds Can we get an sample of UB claims in Allowed amount .
is this a good example ?
"out_of_network":[{
"name": "XYZ",
"billing_code_type": "RC",
"billing_code_type_version": "2020",
"billing_code": "0121",
"description": Medical/surgical/GYN",
"allowed_amounts": [{
"tin": {
"type": "ein",
"value": "1234567890"
},
"service_code": ["01", "02", "03"],
"billing_class": "institutional",
"payments": [{
"allowed_amount": 25.00,
"providers": [{
"billed_charge": 50.00,
